virtfs-ext3/virtfs-ext3.gemspec

```diff
@@ -0,0 +1,36 @@
+# coding: utf-8
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'virtfs/ext3/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = "comcorderfs"
+  spec.version       = VirtFS::Ext3::VERSION
+  spec.authors       = ["Richard Oliveri", "Mo Morsi"]
+  spec.email         = ["roliveri@redhat.com"]
+
+  spec.summary       = "A ext3 based filesystem module for VirtFS"
+  spec.description   = %q{
+  }
+  spec.homepage      = ""
+  spec.license       = "MIT"
+
+  # Prevent pushing this gem to RubyGems.org by setting 'allowed_push_host', or
+  # delete this section to allow pushing this gem to any host.
+  if spec.respond_to?(:metadata)
+    spec.metadata['allowed_push_host'] = "TODO: Set to 'http://mygemserver.com'"
+  else
+    raise "RubyGems 2.0 or newer is required to protect against public gem pushes."
+  end
+
+  spec.files         = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
+  spec.bindir        = "exe"
+  spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
+  spec.require_paths = ["lib"]
+
+  spec.add_runtime_dependency "virtfs"
```

Comment:
If this gem depends on the ext3 gem, then there should probably be a depedency line here.